packages/component-library/src/components/plmg-tooltip/plmg-tooltip.scss

```diff
@@ -0,0 +1,243 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Common styles to all shadow-DOM components
+// Remove this section if your component does not use shadow DOM
+// -------------------------------------------------
+:host {
+  display: block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+// Variables
+// ---------------------------------------------
+
+$arrow-width: 6px;
+
+// ------------- END Variables -----------------
+
+// Mixins
+// ------------------------------
+
+@mixin background-color {
+  &.neutral {
+    background-color: tokens.$plmg-color-background-neutral-dark;
+    border-color: tokens.$plmg-color-background-neutral-dark;
+    color: tokens.$plmg-color-text-neutral-dark;
+  }
+  &.primary {
+    background-color: tokens.$plmg-color-background-primary-strong-active;
+    border-color: tokens.$plmg-color-background-primary-strong-active;
+    color: tokens.$plmg-color-text-neutral-inverted;
+  }
+}
+
+@mixin arrow-color {
+  &.neutral::after {
+    border-color: tokens.$plmg-color-background-neutral-dark;
+  }
+
+  &.primary::after {
+    border-color: tokens.$plmg-color-background-neutral-dark;
+  }
+}
+
+// ------------- END Mixins ------------------------
+
+// Silent Classes
+// ------------------------------------------------
+
+%arrow {
+  &::after {
+    background-color: transparent;
+    content: '';
+    display: block;
+    position: absolute;
+    width: 0;
+    height: 0;
+    border: $arrow-width solid transparent;
+    transform: translate(-50%, 0);
+  }
+}
+
+// ------------- END Silent Classes ------------------------
+
+// Component style
+// ------------------------------------------------
+
+.plmg-tooltip {
+  visibility: hidden;
+  position: relative;
+  display: block;
+  font-family: tokens.$plmg-font-family-sans;
+  font-style: normal;
+  font-weight: tokens.$plmg-font-weight-regular;
+  font-size: tokens.$plmg-font-size-x0_75;
+  line-height: tokens.$plmg-font-lineheight-loose;
+  padding: tokens.$plmg-spacing-x0-25 tokens.$plmg-spacing-x0-5;
+  border-radius: tokens.$plmg-border-radius-s;
+  text-align: center;
+  height: auto;
+  width: fit-content;
+  max-width: 150px;
+  overflow-wrap: break-word;
+
+  &.visible {
+    visibility: visible;
+  }
+
+  @include background-color;
```

Comment:
Since you're using this mixin only once, it's better to inline its content here and remove the mixin